Source/ThirdParty/bitmap-sdf.js

```diff
@@ -0,0 +1,148 @@
+/*
+* https://github.com/dy/bitmap-sdf
```

Comment:
Please add this to the third-party section of LICENSE.md.

Reply:
Done in  75f1da1b8e0ac96064b659a16077f94880e44fcf